src/frontend/src/styles/main.css

```diff
@@ -33,6 +33,21 @@ body {
   width: 100vw;
 }
 
+footer {
+  padding: 2em;
+}
+
+.powered-by-link {
+  padding: 0.5em;
+  display: inline-block;
+}
+
+.powered-by-link:hover,
+.powered-by-link:focus {
+  background: var(--grey-050);
```

Comment:
you can add a small transition to make it smooth, something like

```
.powered-by-link {
    transition: background 0.2s;
```

Reply:
Nice! Added.